### PR TITLE
feat(CORSO-52): change service visibility

### PR DESCRIPTION
**Pull request by Mila5847**
JIRA: link to jira ticket
## Context:
What is the ticket about and why are we doing this change.
## Changes
What are the various changes and what other modules do those changes effect.
This can be bullet point or sentence format.
## Before and After UI (Required for UI-impacting PRs)
If this is a change to the UI, include before and after screenshots to show the differences.
If this is a new UI feature, include screenshots to show reviewers what it looks like. 
## Dev notes (Optional)
- Specific technical changes that should be noted
## Linked pull requests (Optional)
- pull request link

### CODE REVIEW

**Review by KarinaSofia — APPROVED** [review]

File: corso_website_springboot/src/main/java/com/corso/springboot/Service_Subdomain/datamapperlayer/ServiceResponseMapper.java [hunk 1]
```diff
@@ -12,6 +12,7 @@
 public interface ServiceResponseMapper {
 
     @Mapping(expression = "java(service.getServiceIdentifier().getServiceId())", target = "serviceId")
+    @Mapping(expression = "java(service.isActive())", target = "isActive")
```
Comment (DylanBrass) [inline corso_website_springboot/src/main/java/com/corso/springboot/Service_Subdomain/datamapperlayer/ServiceResponseMapper.java:15]: Why was this line required ?

File: corso_website_springboot/src/main/java/com/corso/springboot/Service_Subdomain/presentationlayer/ServiceController.java [hunk 2]
```diff
@@ -24,45 +24,56 @@ public ServiceController(ServiceService serviceService) {
         this.serviceService = serviceService;
     }
 
-    @Cacheable(value = "service", key = "#serviceId")
+    //@Cacheable(value = "service", key = "#serviceId")
```
Comment (DylanBrass) [inline corso_website_springboot/src/main/java/com/corso/springboot/Service_Subdomain/presentationlayer/ServiceController.java:27]: Remind me to fix this issue.

**Review by DylanBrass — APPROVED** [review]
Looks good

File: corso_website_springboot/src/main/java/com/corso/springboot/Service_Subdomain/datamapperlayer/ServiceResponseMapper.java (hunk 1 above)
Comment (Mila5847) [reply corso_website_springboot/src/main/java/com/corso/springboot/Service_Subdomain/datamapperlayer/ServiceResponseMapper.java:15]: Because there was a warning that the property was unmapped.
